WalletWasabi/WabiSabi/Client/CoinJoin/Manager/CoinJoinManager.cs

```diff
@@ -229,7 +250,7 @@ async Task<IEnumerable<SmartCoin>> SanityChecksAndGetCoinCandidatesFunc()
 				return coinCandidates;
 			}
 
-			var coinJoinTracker = await coinJoinTrackerFactory.CreateAndStartAsync(walletToStart, startCommand.OutputWallet, SanityChecksAndGetCoinCandidatesFunc, startCommand.StopWhenAllMixed, startCommand.OverridePlebStop).ConfigureAwait(false);
+			var coinJoinTracker = await coinJoinTrackerFactory!.CreateAndStartAsync(walletToStart, startCommand.OutputWallet, SanityChecksAndGetCoinCandidatesFunc, startCommand.StopWhenAllMixed, startCommand.OverridePlebStop).ConfigureAwait(false);
```

Comment:
Do not use the `!` because there can not exist a `coinJoinTrackerFactory` if there is no coordinator. Please take a look at: https://github.com/WalletWasabi/WalletWasabi/commit/dc71b6fd9736922fd5bc31123f57c6384fb17682